docs/src/developing/programming-model/runtime.md

```diff
@@ -115,11 +115,11 @@ Compute Budget instructions don't require any accounts and don't consume any
 compute units to process.  Transactions can only contain one of each type of
 compute budget instruction, duplicate types will result in an error.
 
-The `ComputeBudgetInstruction::request_units` function can be used to create
+The `ComputeBudgetInstruction::set_compute_unit_limit` function can be used to create
```

Comment:
I was pondering this change on Friday, either way this PR goes using `set` is a good idea and now is the time to make that change.  `set_max_units` or `set_compute_unit_limit`